### PR TITLE
Update to Rollbar 1.1.x

### PR DESCRIPTION
**Pull request by skjb**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,8 +11,14 @@
             "email": "andrei@baibaratsky.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
```
Comment (baibaratsky) [inline composer.json]: Why to use https://asset-packagist.org here?

**Review by baibaratsky — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (skjb) [reply composer.json]: I wasn't able to do a `composer install` without it, I kept getting a dependency error, but it may not be necessary

Comment (baibaratsky) [reply composer.json]: I think that wasn’t because of the rollbar package. Please remove these lines and I’ll merge the PR.

Comment (skjb) [reply composer.json]: Done

**Review by baibaratsky — APPROVED** [review]

**Comment by Borales** [comment]
@baibaratsky hi!

could you please make a release with this change?

**Comment by baibaratsky** [comment]
Sorry for delay
https://github.com/baibaratsky/yii2-rollbar/releases/tag/v1.6.0

**Comment by Borales** [comment]
@baibaratsky thanks!

**Pull request merged**